contrib/devtools/symbol-check.py

```diff
@@ -244,11 +259,23 @@ def check_PE_subsystem_version(filename) -> bool:
         return True
     return False
 
+def check_ELF_interpreter(filename) -> bool:
+    binary = lief.parse(filename)
+    elf = pixie.load(filename)
```

Comment:
using both lief and pixie seems overkill here, please choose one :smile:

Reply:
I would opt for `lief` as we are moving that direction #22392.

Reply:
I explained this in the commit message:

```
Code reviewers: You may see that both lief and pixie are used for
check_ELF_interpreter. That's because at the time of writing:
  - lief does not recognize riscv64 yet (fixed by fanquake in e83f812,
    not yet in a release)
  - pixie does not know how to extract the interpreter
```

Let me know if I'm missing some functionality here!

Reply:
Oops, I missed that commit message, sorry.